### ISSUE TITLE
Point Log load records

### ISSUE THREAD

**Issue by mcdeimon** [VidYen/VYPS-Plugin#116]
I think it would be nice to not load all the records in the Point Log from the administration area on a single page. It would be nice to show 590 results per page or something like that. Now every time you access the Point Log you have to load all the records and that consumes resources unnecessarily.

Thanks!

**Comment by VidYen** [comment]
Yeah. I had thought of that but haven’t implemented it. You can use public
log somewhere and not make it public unless you want to make it public.

**Comment by VidYen** [comment]
And yes I do plan on adding it to the admin page as my own test servers
have quite a log now. :)

**Comment by mcdeimon** [comment]
Hi. Now I see that my server has a hard time loading the records from the administrator.

**Comment by VidYen** [comment]
@mcdeimon

Hey I just rehauled the whole point log system. Mostly on the admin side (and some minor stuff on the public side)

Let me know if it works for you or at least better. For the user log, I have to work with the nonce issue and the public log function doesn't have that built in. So I am going to just leave it at the last 50 for now.

Before it was everyone on the list so I guess its better than nothing.

I need to add search and other logs. However, I think I am going to save that for 1.8.2

I think 1.8.1 actually does a lot of good fixes so I may push it up after more testing.
